rcl/test/rcl/test_service.cpp

```diff
@@ -202,12 +204,14 @@ TEST_F(CLASSNAME(TestServiceFixture, RMW_IMPLEMENTATION), test_service_nominal)
     ret = rcl_take_request(&service, &header, &service_request);
     ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
 
+
```

Comment:
Please remove this unrelated whitespace change.

Reply:
done.